build

```diff
@@ -114,6 +117,15 @@ if [ "$use_kms" = 1 ]; then
 	done
 fi
 
+# Default values which can be overriden via 'build.config' file
+tempfs_size=2G
+
+if [[ -f "$SCRIPT_DIR"/build.config ]]; then
```

Comment:
using `$SCRIPT_DIR` instead of current working directory breaks [local dev mode](https://github.com/gardenlinux/builder?tab=readme-ov-file#local-development) as `build.config` will always be taken from the builder repo, not the config repo

Reply:
yes, calling `./build` from anywhere but the config directory is not yet supported. If we want to support this we'll need to patch a few more things: i.e. we'd need to use `$SCRIPT_DIR` everywhere (podman mounts etc) and then use `realpath --no-symlinks` to only resolve the directory of the build script that is called, ignoring if the build script itself is a symlink, or probably better `realpath $(dirname "${BASH_SOURCE[0]}")`, as it resolves the directory name before doing symlink resolve => symlinked build script will not be followed

Reply:
Fine with me, please see if [ab0636c](https://github.com/gardenlinux/builder/pull/55/commits/ab0636c4f642930538ab22601351e97c410123f6) satisfies your request